### ISSUE TITLE
[Feature Request] More elaborate sign commands?

### ISSUE THREAD

**Issue by Haigen** [ForgeEssentials/ForgeEssentials#2177]
He there, loving the mod, using 1.8.9-1.8.5.101. 
Just a query on whether or not it's possible to get signs to display something whilst actually driving a separate command? 

e.g. Sign says: "Click here to become a member" but runs the /feperm user group set Member or something? 
Also is there a way to tell the command to use the player interacting with the sign in the command? 

If there's a better alternative for this solution lemme know. Just trying to set something up for a server so that players have to read a set of rules before being able to interact with certain things in the world.

Cheers.

**Comment by olee** [comment]
You know... for that you'd use a button + command block with a sign above the button.
It's weird to begin with clicking a sign to do something.

Or do you punch signs in the real world? :laughing: 

TL;DR: I guess there are way better ways for this by not using a sign.
If you can find a (good) reason to still have this, tell me.

**Comment by spacebuilder2020** [comment]
Only reason I see to do it is because you can do this kind of stuff with bukkit.  However, perhaps binding a sign to a script would be better.

**Comment by olee** [comment]
Yeah that was my opinion as well.
What you can do though is using the new scripting feature you could actually write a script with an event handler on player interaction to handle players clicking that sign.

I'd say if you really want to trigger some action by clicking a sign with any text without putting a button next to it, you could use that. 
Would require some work though.

I'll close this as a wontfix, because there are other solutions for this (scripting / other specialized mods)
